pkgs/tools/security/onlykey/generate.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix
+
+# XXX: --development is given here because we need access to gulp in order to build OnlyKey.
+exec node2nix --nodejs-10 --development -i pkg.json -c onlykey.nix -e ../../../development/node-packages/node-env.nix --no-copy-node-env
```

Comment:
Can we use a newer node version?
```suggestion
exec node2nix --nodejs-10 --development -i package.json -c onlykey.nix -e ../../../development/node-packages/node-env.nix --no-copy-node-env
```

Reply:
Yep, thanks for the reminder. I moved to using the latest supported by node2nix, `--nodejs-14`.